src/expressions/plyql.pegjs

```diff
@@ -785,7 +808,7 @@ ExpressionMaybeFiltered
 
 
 FunctionCallExpression
-  = fn:Fn OpenParen params:Params CloseParen
+  = fn:Fn OpenParen params:(Expression As / Params) CloseParen
```

Comment:
I am thinking it might not be a good idea to treat cast as a function... or maybe it is?